examples/schoolyard.jl

```diff
@@ -0,0 +1,142 @@
+# # Social networks with LightGraphs.jl
+
+# ![](play.gif)
+
+# Many ABM frameworks provide graph infrastructure for analysing network properties of agents.
+# Agents.jl is no different in that aspect, we have [`GraphSpace`](@ref) for when spatial structure
+# is not important, but connections are.
+
+# What if you wish to model something a little more complex? Perhaps a school yard full of students
+# running around (in space), interacting via some social network. This is precisely the scenario that
+# the [MASON](https://cs.gmu.edu/~eclab/projects/mason/) ABM framework uses as an introductory example
+# in their [documentation](https://cs.gmu.edu/~eclab/projects/mason/manual.pdf).
+
+# Rather than implementing an Agents.jl&ndash;specific graph structure, we can interface with
+# [LightGraphs.jl](https://github.com/JuliaGraphs/LightGraphs.jl): a high class library for managing
+# and implementing graphs, which can be re-used to establish social networks within existing spaces.
+
+# To begin, we load in some dependencies
+
+using Agents, LightGraphs, SimpleWeightedGraphs, SparseArrays, AgentsPlots
```

Comment:
Where in the example do you use SparseArrays?

Also, do you need to use both LightGraphs and SimpleWeightedGraphs? I think only the latter is enough

Reply:
findnz is in sparsearrays

Reply:
add_edge! is in LightGraphs.